### PR TITLE
304 rename geostationary sectors

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist

* [x] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***unit tests*** added and pass for new/modified functionality
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***integration tests*** added and pass for new/modified functionality
  * [x] None required. (explain if checked, delete line if not checked)
* [x] Required ***documentation*** added for new/modified functionality
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***release notes*** added for new/modified functionality
  * [x] None required. (explain if checked, delete line if not checked)

# Related Issues
fixes NRLMMD-GEOIPS/geoips#304

# Testing Instructions
Run geoips/tests/integration_tests/full_test.sh; this will return a 0 upon success.

# Summary
GeoIPS has a branch which renamed geostationary sectors. To adhere to these newly named
sectors, we need to update scripts in this package, and well as the outputs, to use
these newly named sectors.

    modified: tests/scripts/abi_aws.sh
    modified: tests/outputs/abi_aws/Infrared-Test_image/20200918.195020.goes-16.abi.Infrared-Test.goes16.45p56.noaa.10p0.png

### CODE REVIEW

File: docs/source/releases/v1_11_7a0.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+*From issue GEOIPS#304: 2023-08-27, Rename Geostationary Sectors*
+
+GeoIPS has a branch which renamed geostationary sectors. To adhere to these newly named
+sectors, we need to update scripts in this package, and well as the outputs, to use
+these newly named sectors.
+
+::
+
+    modified: tests/scripts/abi_aws.sh
+    modified: tests/outputs/abi_aws/Infrared-Test_image/20200918.195020.goes-16.abi.Infrared-Test.goes16.45p56.noaa.10p0.png
```
Comment (mindyls) [inline docs/source/releases/v1_11_7a0.rst:33]: Again, can we confirm this image did not actually change?  There should be no modifications to the image itself.

**Review by mindyls — DISMISSED** [review]
Again, I'm good with this one *if* we can confirm the image did NOT change at all.  It should have just been moved without changes, but it shows up as deleted and added.

File: docs/source/releases/v1_11_7a0.rst (hunk 1 above)
Comment (mindyls) [reply docs/source/releases/v1_11_7a0.rst:33]: I updated it, now it is renamed without changes.

**Review by mindyls — APPROVED** [review]
This looks good to me

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**